Resources/Private/JavaScript/Host/Containers/ContentCanvas/InlineUI/MarkActiveNodeAsFocused/index.js

```diff
@@ -0,0 +1,32 @@
+import {Component} from 'react';
+import {connect} from 'react-redux';
+import {CR} from 'Host/Selectors/index';
+import {$transform} from 'plow-js';
+import style from './style.css';
+import {node as NodePropType} from 'Shared/PropTypes/CR/Node/index';
+
+@connect($transform({
+    focusedNode: CR.Nodes.focusedSelector
+}))
+export default class MarkActiveNodeAsFocused extends Component {
```

Comment:
Why does it need to be a react component at all, and not just plain JS module, if it's not rendering anything, but rather directly manipulating DOM?